public/js/use.js

```diff
@@ -288,7 +288,7 @@ function setupUse(params) {
           var image = new Image();
           image.src = reader.result;
           image.onload = function () {
-            var resizedImage = (data.files[0]['size'] > 2000000) ? resize(image, 800) : this.src;
+            var resizedImage = resize(image, 800);
```

Comment:
I can have images that are bigger than 2000x2000 but still under 2 MB.  Those images should not be transformed because the transformation could affect the recognition. 
The next image is 1920 × 1080 and 1.7 MB.
![cbs_big_brother_1802_clean_thumb_master](https://cloud.githubusercontent.com/assets/313157/21482500/7435abf8-cb53-11e6-947d-124bba62f900.jpg)

We probably need to separate `exif` and resize as two different functions, and only apply the second one when
``` js
data.files[0]['size'] > 2000000
```
Like we used to.